### PR TITLE
Mobile Menu

### PR DESCRIPTION
**Pull request by looh16**
1. Added a menu for the mobile version.
2. allow clicking on any menu item in the mobile version to redirect to the section of the page related to the item selected in the menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,8 +29,16 @@
...
                 </nav>
 
-                <input type="checkbox" id="checkbox_toggle"/>
-                <label for="checkbox_toggle" class="toggle"><img src="images/Union.svg" alt="Menu Icon"></label>
+                <div id="hamburger" onclick="toggleHamburgerMenu(this)">
+                    <div class="first_line"></div>
+                    <div class="second_line"></div>
+                    <div class="third_line"></div>
+                    <ul class="mobile-menu">
+                      <li><a class="nav-link" href="#top">Portoflio</a></li>
```
Comment (danmainah) [inline index.html:37]: - [ ] KIndly check on your portfolio name it's misspelled here.
- [ ] KIndy check on your design, it currently doesn't match the design given, below is the design given:
image

**Review by danmainah — COMMENTED** [review]
Hi @looh16 and @SamuelTTW , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used


# Required Changes ♻️

- [ ] Kindly ensure to setup your linter files correctly, for this milestone you need to set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 2]
```diff
@@ -28,8 +29,16 @@
...
-                <label for="checkbox_toggle" class="toggle"><img src="images/Union.svg" alt="Menu Icon"></label>
+                <div id="hamburger" onclick="toggleHamburgerMenu(this)">
+                    <div id="first" class="first_line"></div>
+                    <div id="second" class="second_line"></div>
+                    <div id="third" class="third_line"></div>
+                    <ul class="mobile-menu">
+                      <li><a class="nav-link" href="#top">Portfolio</a></li> 
+                      <li><a class="nav-link" href="#about">About</a></li> 
+                      <li><a class="nav-link" href="#contact-form">Contact</a></li>
+                    </ul>
```
Comment (ishimwezachee) [inline index.html:40]: - [ ]  As previous reviewer said, you design should match with the design provided from figma ,please implement it 

Your design          |  Figma design
:-------------------------:|:-------------------------:
image  |  image

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @looh16 and @SamuelTTW  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

File: css/styles.css [hunk 3]
```diff
@@ -13,83 +13,72 @@
...
-nav {
-  display: flex;
-  justify-content: space-between;
+.nav-link:hover {
+  color: dodgerblue;
 }
 
-ul {
-  list-style: none;
+.hamburger {
```
Comment (Ismailco) [inline css/styles.css:64]: - [ ] As the previous reviewer mentioned you should duplicate the mobile menu design, kindly add blur to the background, and use the same padding/margin as in the Figma design so your design meets the requirements

|Your design|Figma design|
|-|-|
|Screenshot_20220621_194548|Screenshot_20220621_195352|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @looh16,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

#### Note: I see that you had the same changer requested multiple times, if you need help don't hesitate to contact me on slack or comment on this review. Happy coding 😄

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css [hunk 4]
```diff
@@ -13,83 +13,78 @@
...
+  padding: 0 24px;
 }
 
-input[type=checkbox] {
-  display: none;
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
```
Comment (Mre55) [inline css/styles.css:41]: **[OPTIONAL]** I would hight suggest here giving `height: 100vh` that way the `mobile menu` will cover the entire height of the screen.

**Review by Mre55 — APPROVED** [review]
**Hi @looh16,**

# Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 



- [ ] Check my OPTIONAL suggestion below 👇 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**